invenio_app_ils/holidays/tasks.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Holidays tasks."""
```

Comment:
Can you please add logging to both tasks here? It would be great to have JSON formatted logging as done in `ReferencedRecordsIndexer` class.
The idea is to be able to understand what happened in case of an error by checking logs:

- which past exceptions were automatically deleted? (if you get a support email asking "my previous exceptions disappeared, can you tell me which ones were there?")
- for which active loans we have automatically change the end date? What are the previous and the new end date? (imagine you get a support ticket asking "for the loan of Mr. Mario, the end date changed" and you need to find in the logs what happened)